subsys/net/l2/ieee802154/ieee802154.c

```diff
@@ -166,8 +161,8 @@ static bool ieeee802154_check_dst_addr(struct net_if *iface,
 		 * set to TRUE, an 64-bit extended unique identifier (EUI-64)
 		 * group address.
 		 */
-		if (memcmp(dst_plain->addr.ext_addr, ctx->ext_addr,
-					IEEE802154_EXT_ADDR_LENGTH) != 0) {
+		if (memcmp(dst_plain->addr.ext_addr, ctx->ext_addr, IEEE802154_EXT_ADDR_LENGTH) !=
+		    0) {
```

Comment:
While I totally agree this has larger repercussions - I therefore opened an RFC for better visibility of the question... see #48616

Reply:
@jukkar, is it ok, to leave this as-is in this PR and prepare a separate PR for the general formatting change as proposed by @gmarull in #48616? This would decouple the more general discussion from this changeset.

Reply:
Fixed manually for the moment being (breaks automatic formatting) as this shouldn't hold up the PR. Remainder to be discussed in #48616 and fixed in a separate PR.